src/pyobo/api/utils.py

```diff
@@ -46,3 +50,40 @@ def get_version(prefix: str) -> Optional[str]:
         return data["version"]
 
     return None
+
+
+def get_version_pins():
```

Comment:
1. you can put a `lru_cache(1)` on the outside of this since it won't change during runtime
2. type annotate everything, please